core/variables.js

```diff
@@ -67,11 +67,11 @@ Blockly.Variables.allUsedVarModels = function(ws) {
     }
   }
   // Flatten the hash into a list.
-  var variableList = [];
+  var VariableList = [];
```

Comment:
Curious, why the capitalization of `variableList`?

Reply:
Cuz typos are my best friends :P hehe had some accidental renamings and looks like I didn't quite fix this one. Thanks for the catch!